packages/cfd/src/Components/cfd-download-container.tsx

```diff
@@ -22,7 +22,7 @@ type TMobileDownloadProps = {
     platform: TCFDDashboardContainer['platform'];
 };
 
-const mobileDownloadLink = (platform: string, type: 'ios' | 'android' | 'huawei') => {
+export const mobileDownloadLink = (platform: string, type: 'ios' | 'android' | 'huawei') => {
```

Comment:
Why are you exporting this and duplicating the same function in `derivx-trade-modal.tsx` file?